### DISCUSSION TITLE
Project transfer

### DISCUSSION THREAD

**Discussion by pariweshh** [vercel/community#1942] [Help] [answered]
### Summary

Hi, i wanted to know if there is a way to transfer an entire project to one vercel account from another. When I go to transfer under the settings, the only option I am able to select is the 'Create new team' option.

**Comment by swarnava** [comment, marked as answer]
Hi,

You can only transfer project between your teams or your own account. For separate "personal / hobby" account, you need to setup the project from scratch.
